### ISSUE TITLE
Is autopagination supported for batch requests

### ISSUE THREAD

**Issue by GCRoberts** [facebook/facebook-java-business-sdk#335]
#### Observed Results:

  * What happened?  This could be a description, log output, etc.

#### Expected Results:

  * What did you expect to happen?

**Comment by GCRoberts** [comment]
When doing batch requests using Graph API, you get back 25 results, plus a "next" link in the paging that allows you to get the next 25 results.  This also happens when doing non-batched.  In the SDK, you can specify "autopagination" when you execute the command.  However, there is no "autopagination" option available when you do an "addToBatch".  So when you make a request that has more than 25 results, there doesn't appear to be a way to iterate beyond the first 25 results.  This makes batch requests pretty much useless in the SDK.  Are we missing something?  It seems somewhat straightforward to implement when not using batch request.

**Comment by GCRoberts** [comment]
Turns out that I was missing something after all.  The batch response is a List<APIResponse> and I was trying to set the autopagination on that list.  What I needed to do is typecast the individual APIResponse as APINodeList<Lead>.  Once I did that, I was able to apply the "withAutoPaginationIterator(true)" to that object.  After doing that, the iteration will give all results, well beyond 25 entries.  Additional requests are made for each set of 25, but that's hidden behind the scenes of the iteration.   It is important to realize that if the first request works and you're able to start the iteration, that does not guarantee that following requests won't fail.
